qcodes/dataset/descriptions/dependencies.py

```diff
@@ -488,7 +488,7 @@ def __repr__(self) -> str:
                f"standalones={self.standalones})")
         return rep
 
-    def __eq__(self, other):
+    def __eq__(self, other: Any) -> bool:
 
         def sorter(inp: Any) -> List[Any]:
```

Comment:
shouldn't this one actually `ParamSpecBase`, not `Any`?

Reply:
Nope you are allowed to compare this with anything (but it returns False naturally)

Reply:
i meant the `Any` in the `sorter` method

Reply:
OK :) I think this is correct